README.md

```diff
@@ -1,18 +1,21 @@
 # Dynatrace-automation-tools
 
+> Note: If you are reading this in Docker Hub, please check the full documentation in the Github Repo [Dynatrace Automation Tools](https://github.com/dynatrace-ace/dynatrace-automation-tools)
+
 This CLI allows automating different Dynatrace use-cases to be used with CI/CD or automation platforms. The current implemented use cases:
 
-1. [Site Reliability Guardian Automation](docs/Site-Reliability-Guardian/SRGAutomation.md)
+1. [Site Reliability Guardian Automation](https://github.com/dynatrace-ace/dynatrace-automation-tools/tree/main/docs/Site-Reliability-Guardian/SRGAutomation.md)
+1. [Dynatrace Events](https://github.com/dynatrace-ace/dynatrace-automation-tools/tree/main/docs/Events/Readme.md)
```

Comment:
Just a quick note, if relative paths were used, refs would also work within a feature branch. Is there a specific reason these are absolute paths?

Reply:
done

Reply:
I remember why i was using the absolute links, It's due to the same readme file being reused in the docker image description. Absolute links work but relative ones don't since the url is https://hub.docker.com/r/dynatraceace/dt-automation-cli.